src/Utility/Filter.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @since         5.0.1
```

Comment:
```suggestion
 * @since         5.1.0
```
Patch releases only contain bug fixes, new features need to go in the next minor release.

Reply:
thx. Fixed in https://github.com/cakephp/cakephp/pull/17295/commits/01ec52d229047f93d0a1d4a39e3eb086593a79ce